runners/spark/src/main/java/org/apache/beam/runners/spark/stateful/SparkTimerInternals.java

```diff
@@ -92,6 +93,13 @@ public static SparkTimerInternals forStreamFromSources(
         slowestLowWatermark, slowestHighWatermark, synchronizedProcessingTime);
   }
 
+  /** Build a global {@link TimerInternals} for all feeding streams.*/
+  public static SparkTimerInternals forStreamFromSources(
```

Comment:
Changed this specific method to `global`, since it really provides a "global" `SparkTimerInterals`.